src/components/Synthetics/TradeFeesRow/TradeFeesRow.tsx

```diff
@@ -57,18 +58,47 @@ export function TradeFeesRow(p: Props) {
   const shouldShowRebate = p.shouldShowRebate ?? true;
   const rebateIsApplicable = shouldShowRebate && p.positionFee?.deltaUsd.lt(0) && p.feesType !== "swap";
 
+  const [fullPositionPriceImpact, hasRebates] = mergePositionPriceImpactWithPriceImpactDiff(
+    p.feesType,
+    p.positionPriceImpact,
+    p.priceImpactDiff
+  );
+  const rebatesMessage = hasRebates ? (
+    <Trans>
+      Price Impact Rebates for closing trades are claimable under the Claims tab.{" "}
+      <ExternalLink newTab href="https://docs.gmx.io/docs/trading/v2/#price-impact-rebates">
+        Read more
+      </ExternalLink>
+      .
+    </Trans>
+  ) : undefined;
+
   const feeRows: FeeRow[] = useMemo(() => {
-    const positionPriceImpactRow = p.positionPriceImpact?.deltaUsd.abs().gt(0)
+    const positionPriceImpactRow = fullPositionPriceImpact?.deltaUsd.abs().gt(0)
       ? {
           id: "positionPriceImpact",
           label: (
             <>
               <div className="text-white">{t`Position Price Impact`}:</div>
-              <div>({formatPercentage(p.positionPriceImpact.bps.abs())} of position size)</div>
+              <div>({formatPercentage(fullPositionPriceImpact.bps.abs())} of position size)</div>
+            </>
+          ),
+          value: formatDeltaUsd(fullPositionPriceImpact.deltaUsd),
+          className: fullPositionPriceImpact.deltaUsd.gte(0) ? "text-green" : "text-red",
+        }
+      : undefined;
+
+    const priceImpactDiffUsd = p.priceImpactDiff?.deltaUsd.abs().gt(0)
+      ? {
+          id: "priceImpactDiff",
+          label: (
+            <>
+              <div className="text-white">{t`Price Impact Rebates`}:</div>
+              <div>({formatPercentage(p.priceImpactDiff.bps.abs())} of position size)</div>
             </>
           ),
-          value: formatDeltaUsd(p.positionPriceImpact.deltaUsd),
-          className: p.positionPriceImpact.deltaUsd.gte(0) ? "text-green" : "text-red",
+          value: formatDeltaUsd(p.priceImpactDiff.deltaUsd),
+          className: p.priceImpactDiff.deltaUsd.gte(0) ? "text-green" : "text-red",
```

Comment:
We can use `getPositiveOrNegativeClass` just to be consistent

Reply:
we don't have it in this file, also behavior of getPositiveOrNegativeClass is slightly different, I've adjusted it with `zeroValue` arg:
```
export function getPositiveOrNegativeClass(value?: BigNumber, zeroValue: "" | "text-red" | "text-green" = ""): string {
  if (!value) {
    return "";
  }
  return value.isZero() ? zeroValue : value.isNegative() ? "text-red" : "text-green";
}
```